ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/BadBlockCause.java

```diff
@@ -64,20 +53,8 @@ public String getDescription() {
     return description;
   }
 
-  public Optional<Throwable> getException() {
-    return exception;
-  }
-
   @Override
   public String toString() {
-    return "BadBlockCause{"
-        + "reason="
-        + reason
-        + ", description='"
-        + description
-        + '\''
-        + ", exception="
-        + exception
-        + '}';
+    return "BadBlockCause{" + "reason=" + reason + ", description='" + description + '\'' + '}';
```

Comment:
```suggestion
    return "BadBlockCause{reason=" + reason + ", description='" + description + "\}";
```

Reply:
Yeah - this method is kind of inconsistent and not very readable.  Regenerated using `MoreObjects`.